src/app/_shared-components/SetIdentity/SetIdentityFees/SetIdentityFees.tsx

```
@@ -7,6 +7,10 @@ import { BN } from '@polkadot/util';
 import { Separator } from '@/app/_shared-components/Separator';
 import { Button } from '@/app/_shared-components/Button';
 import { useTranslations } from 'next-intl';
+import { getCurrentNetwork } from '@/_shared/_utils/getCurrentNetwork';
```

ESLint

[error] 'getCurrentNetwork' is defined but never used.

(@typescript-eslint/no-unused-vars)